app/controllers/candidates/home_controller.rb

```diff
@@ -3,4 +3,6 @@ def index
     @applications_deactivated = \
       Rails.application.config.x.candidates.deactivate_applications
   end
+
+  def what_to_expect; end
```

Comment:
Could we change the URL path to school_experience_guide_for_candidates ? @leoapost

Reply:
The final URL will look like:
```https://schoolexperience.education.gov.uk/candidates/school_experience_guide_for_candidates```

I was thinking we could make it shorter since the `school experience` and `candidates` are already part of the url. Perhaps something like:
`https://schoolexperience.education.gov.uk/candidates/guides`

@B-Bonner @MylesJarvis  any thoughts?

Reply:
Yeah, good points @leoapost. Who could we speak to about it?

Reply:
Maybe we could ask Shivani? @MylesJarvis any thoughts?

Reply:
I wonder whether we compromise on 'guide_for_candidates'. As you say, we've got the school experience reference in the URL.

That should keep SEO peeps happy.

Reply:
> 

Yep, sounds good if that's allowed. @leoapost ?

Reply:
I've pushed the changes